### DISCUSSION TITLE
Grafana Version

### DISCUSSION THREAD

**Discussion by AlHood77** [GoogleCloudPlatform/prometheus-engine#333] [Q&A] [answered]
Was reading through issue #247 and noticed that it is now fixed. Just wondering what Grafana versions people are using with Managed Prometheus and not having any issues?

**Comment by pintohutch** [comment, marked as answer]
I'm not sure we have any quantitative stats on this @lyanco?

As a data point, we peg a version in our [example manifest](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/c5b2885baad4ec209a4f9a8c6dab722692c21dd7/examples/grafana.yaml#L45) that folks may be using.

**Comment by lyanco** [comment]
I haven't heard any systemic complaints about newer versions of Grafana... but if you see any problems, please let us know.

I think they might have changed a call or two to use label_values() without first specifying a metric, which might cause issues with autocomplete or lookups. If that's the case, we'll fix it, just give us a holler.
